instrumentation/pg/test/opentelemetry/instrumentation/pg/instrumentation_test.rb

```diff
@@ -361,5 +373,21 @@
         _(span.attributes['net.peer.port']).must_equal port.to_i if PG.const_defined?(:DEF_PORT)
       end
     end
+
+    def self.load_fixture
+      data = File.read("#{Dir.pwd}/test/fixtures/sql_table_name.json")
+      JSON.parse(data)
+    end
+
+    load_fixture.each do |test_case|
+      name = test_case['name']
+      query = test_case['sql']
+
+      define_method(:"test_sql_obfuscation_#{name}") do
```

Comment:
I think this might've been a copy-paste miss! Maybe "test_sql_table_name_" instead?